Mathlib/Probability/Distributions/Gaussian.lean

```diff
@@ -426,14 +451,136 @@ theorem mgf_gaussianReal (hX : p.map X = gaussianReal μ v) (t : ℝ) :
   rw [← mgf_id_map hX_meas, ← complexMGF_ofReal, hX, complexMGF_id_gaussianReal, mul_comm μ]
   norm_cast
 
+theorem mgf_id_gaussianReal :
```

Comment:
could you keep this name for the lemma where you would use `id`, and rename this one with `fun_id`? Same thing below several times.